src/App.scss

```diff
@@ -0,0 +1,18 @@
+@use './styles/global';
+@import './styles/variables.scss';
+
+.app {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: var(--padding);
+  font-family: var(--font-family);
+}
+
+h1 {
```

Comment:
**suggestion:** I would avoid adding style to html elements directly in your styling and just a css class.

Reasoning: If you ever used a custom css library, your own styling directly of html elements might conflict with it based on css specification.

Reply:
Helpful. I`ll keep this in mind